scripts/create_and_deploy_recipe.sh

```diff
@@ -73,6 +74,9 @@ while [ "$1" != "" ]; do
             shift
             BUCKET="$1"
             ;;
+        --gendbfiles)
```

Comment:
I think that the gendbfiles functionality need to be implied.
i.e. if the recipe contains pre-generated block generation, then we should always use these.

If you want to add an option to override that, (i.e. disable that ), that's fine, of course.